handbook/communicating.md

```diff
@@ -0,0 +1,26 @@
+# Other communication
+
+:::{admonition} NEW!!!
+:class: warning
+This content is new and should be reviewed!
+:::
+
+GitHub is a fantastic platform for collaborating on open source software, but it can't do *everything*.  Additionally, we work with other people or institutions that use *different* platforms for collaboration, teamwork, and communication.  While Xdev uses GitHub for as much as we can, we acknowledge that sometimes you have to use something other than GitHub from time to time.
+
+## Email
+
+Email should be used sparingly, but there are times when it is necessary.  Sending inviations to Zoom or Google Meet calls sometimes requires knowing a teammate's individual email address, which results in an email notification of the newly scheduled event.  Email is also a convenient mechanism for people *outside* of the team to communicate with us, in which case responses over email are obviously appropriate.
+
+Communication directly between team members with email should be discouraged, though.  The only time when such communication is appropriate is when you need to reference another email (e.g., replying or forwarding) for discussion, or you need to reference a Zoom or Google Meet event for which you were notified via email.
+
+These rules apply for both your individual work email addresses *and* the Xdev Team email address (`xdev@ucar.edu`).
+
+## Zulip
+
+Zulip is primarily useful for replying to people who are *outside the team* who have questions that the team can answer about Python, the Pangeo software stack, and general application questions regarding analysis.  The Team should monitor Zulip on a semi-regular basis, once or twice a day.  Fortunately, traffic on Zulip is not so great as to require more than that, even for a team our size!
+
+Zulip can also be useful for one-on-one communication between teammates.  Use Zulip for this rather than email, as much as possible.  Also, use Zulip for direct communication with another person primarily as a means of keeping other communication channels "less noisy," even if the communication specifically doesn't have anything terribly private in it.
+
+## Gather Town
```

Comment:
I thought we couldn't use Gather town since it wasn't/isn't approved by UCAR

Reply:
Yeah.  See my comment below to @mnlevy1981.  I'm commenting this out for now.  But I'm going to ask for special permission to use it.